packages/frontend-2/components/viewer/comments/Comments.vue

```diff
@@ -62,7 +62,10 @@
       <div v-if="commentThreads.length === 0" class="pb-4">
         <ProjectPageLatestItemsCommentsEmptyState
           small
-          :show-button="canPostComment"
+          :show-button="canPostComment && hasSelectedObjects"
+          :text="
+            hasSelectedObjects ? undefined : 'Select an object to start collaborating'
```

Comment:
<img width="382" alt="image" src="https://github.com/user-attachments/assets/581da5ec-1f1d-4200-aea1-66949d439847">
@benjaminvo feel free to update this copy